src/axom/sidre/core/View.hpp

```diff
@@ -431,6 +431,21 @@ class View
 
   //@}
 
+  /*!
+   * \brief Reshape the array without changing its size.
+   *
+   * The View state must be either BUFFER or EXTERNAL.
+   * The new shape must have the same size as the current.
```

Comment:
Clarification -- do you think we should clarify what "same size" mean w.r.t. shapes (especially if the View originally did not have a shape)? Or do you think it will be clear to users?

Perhaps it would help to mention how "same size" is related to `View::getNumElements()`?

Reply:
Also, do have/need a way to remove/clear a shape from a View, i.e. to return it to a 1D array?

Reply:
Referenced `getNumElements()` as size.  There is a way to clear a shape from View:
`IndexType s = v->getNumElements(); v->reshapeArray(1, &s);`  I'll add it to the test.